package.json

```diff
@@ -10,8 +10,10 @@
   "scripts": {
     "build": "tsc -p ./",
     "prepare": "npm run build",
-    "lint": "npx eslint src/**/*.ts",
-    "lint:fix": "npx eslint --fix src/**/*.ts",
+    "lint": "npx eslint 'src/**/*.ts'",
```

Comment:
[suggestion] I suggest adding `tsc --noEmit` to the lint step as well. Not sure if `eslint` actually covers compilation part.